test/testcafe/framework/page-objects/SelectAuthenticatorPageObject.js

```diff
@@ -9,6 +9,7 @@ const factorSelectButtonDiv = `${factorListRowSelector} .authenticator-button`;
 const factorSelectButtonSelector = `${factorListRowSelector} .authenticator-button .button`;
 const skipOptionalEnrollmentSelector = '.authenticator-list .skip-all';
 const CUSTOM_SIGN_OUT_LINK_SELECTOR = '.auth-footer .js-cancel';
+const CUSTOM_OTP_BUTTON_SELECTOR = '.authenticator-list .authenticator-row:nth-child(11) .authenticator-button a';
```

Comment:
using this filter `authenticator-row:nth-child(11)` for selector would always work if the list grows ?

Reply:
Yes, As long as we keep adding items to this list.

Reply:
so the test will fail and we need to change the number to pass it ?

Reply:
The test wont break, if we add authenticators to the list. Similar to how we check for authenticators in the list [here](https://github.com/okta/okta-signin-widget/blob/master/test/testcafe/spec/SelectAuthenticatorForVerification_spec.js#L177)